msteams-platform/m365-apps/overview.md

```diff
@@ -80,6 +79,11 @@ You can now define and deploy Outlook Add-ins in [version 1.17 and later](../res
 
 For more information, see [app manifest for Office Add-ins](/office/dev/add-ins/develop/unified-manifest-overview).
 
+### Word, Excel, PowerPoint Add-ins (preview)
+You can now define and deploy Word, Excel, PowerPoint Add-ins in [version 1.20 and later](../resources/schema/manifest-schema-dev-preview.md) of the app manifest.
```

Comment:
This statement makes it sound like WXP add-ins and v1.20 are already GA. How about saying something like:

```
You can now define and deploy Word, Excel, and PowerPoint Add-ins with the [developer preview](../resources/schema/manifest-schema-dev-preview.md) unified manifest for Microsoft 365.
```

Reply:
Looks better, thanks.